file.rb

```diff
@@ -0,0 +1,68 @@
+# swiped from OOD Appkit
+# not currently being used
```

Comment:
Is there a reason to keep this file?

Reply:
I think it should stay until we figure out how we want to deal with downloading the file to be edited and saving the changes which is currently provided as a part of the file explorer app.